pipelines/rj_crm__salesforce_agentforce_api/tasks/notify.py

```
@@ -0,0 +1,187 @@
+) -> None:
+    """
+    Notifica sucesso de uma fase.
+
+    Args:
+        phase_name    : Nome da fase (ex: 'F1 — STDM').
+        rows_by_table : Dict {tabela: qtd_linhas} para resumo.
+        elapsed_seconds: Tempo de execução da fase.
+    """
+    webhook_url = os.getenv("SLACK_WEBHOOK_URL")
+    if not webhook_url:
+        print(f"[NOTIFY] SLACK_WEBHOOK_URL nao configurada — notificacao pulada.")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)